code/modules/food_and_drinks/kitchen_machinery/griddle.dm

```diff
@@ -35,7 +36,7 @@
 	if(default_deconstruction_crowbar(I, ignore_panel = TRUE))
 		return
 	variant = rand(1,3)
-	RegisterSignal(src, COMSIG_ATOM_EXPOSE_REAGENT, .proc/on_expose_reagent)
+	UnregisterSignal(src, COMSIG_ATOM_EXPOSE_REAGENT)
```

Comment:
Why do we need to unregister the signal... for when we're changing the variant of our griddle?

Reply:
Nyoops. Okay Ill feex later tonight.